public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -11,11 +11,11 @@ a#top
     * [Add _i18n-..._ translation attributes](#translate-attributes)
     * [Handle singular and plural](#cardinality)
     * [Select among alternative texts](#select)
-    * [Use the **_ng-xi18n_ extraction tool** to create a translation source file](#ng-xi18n)
-    * [Translate](#translate)
+    * [Create a translation source file with the **_ng-xi18n_ extraction tool**](#ng-xi18n)
+    * [Translate los mensajes de texto](#translate)
```

Comment:
This should stay in English: "Translate text messages".  The Spanish was cute but inconsistent. Let's fix the actual subhead too.